### PR TITLE
adds toggle to fetch extensions from localhost

### PR DESCRIPTION
**Pull request by finagle29**
now you only have to have one xkit installed per browser when developing!

suggestion are appreciated as to what text to show in _preferences

### CODE REVIEW

File: Extensions/xkit_preferences.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //* TITLE XKit Preferences **//
-//* VERSION 5.0.0 **//
+//* VERSION 5.0.1 **//
 //* DESCRIPTION Lets you customize XKit **//
 //* DEVELOPER STUDIOXENIX **//
```
Comment (Wolvan) [inline Extensions/xkit_preferences.js]: Nitpick: Change Developer to new-xkit

**Comment by 0xazure** [comment]
:-1: from me.  Something like this should not be user-facing in XKit itself.  I would rather see it as a build task that [templates the required data](https://github.com/sindresorhus/gulp-template) into `xkit.js`.

File: Extensions/xkit_preferences.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 //* TITLE XKit Preferences **//
-//* VERSION 5.0.0 **//
+//* VERSION 5.0.1 **//
```
Comment (0xazure) [inline Extensions/xkit_preferences.js]: This is adding functionality, so [per semver](http://semver.org/) should be a minor patch bump.

**Comment by blackjackkent** [comment]
Agreed that this should not be a user-facing flag; definitely a useful feature but the chanel strikes me as not the right place for it.

**Comment by nightpool** [comment]
Yes, I would love to put this into the build system. Not a huge fan of
putting it into preferences.

**Comment by finagle29** [comment]
Closing in favor of a new build task.

**Pull request closed without merging**